docs/design.adoc

```diff
@@ -268,27 +268,17 @@ surplus value allows for it. For example:
 Each notifier reporting about deposit liquidation start (`notifyLiquidation`)
 or about a deposit being liquidated outside of the coverage pool
 (`notifyLiquidated`) can be rewarded with COV tokens. Those tokens represent a
-part of the asset pool ownership. The amount of token reward is set by
-the governance. It can be determined either as a fixed amount or as a
-percentage of the total COV supply. Current values of reward parameters can
-be viewed using the following methods of the risk manager:
+part of the asset pool ownership. The amount of token reward is set by the
+governance. The current amounts of rewards can be viewed using the following
+methods of the risk manager:
 
 * For `notifyLiquidation` rewards:
 ** `liquidationNotifierRewardAmount`: determines the fixed amount of COV tokens
 which is granted as reward for reporting about a deposit liquidation process.
-** `liquidationNotifierRewardPercentage`: determines the percentage of the
-total COV supply which is granted as reward for reporting about a deposit
-liquidation process.
 * For `notifyLiquidated` rewards:
 ** `liquidatedNotifierRewardAmount`: determines the fixed amount of COV tokens
 which is granted as reward for reporting about a deposit being liquidated
 outside of the coverage pool.
-** `liquidatedNotifierRewardPercentage`: determines the percentage of the
-total COV supply which is granted as reward for reporting about a deposit
-being liquidated outside of the coverage pool.
-
-If both fixed amount and percentage values are set and bigger than zero,
-the fixed amount reward takes precedence.
```

Comment:
Given that we have only one bullet for each notification, the rendering looks weird:

![image](https://user-images.githubusercontent.com/4712360/124930314-8374ef80-e001-11eb-8120-818435eb1ce9.png)

Can we update it to something like:
```
* For `notifyLiquidation`, `liquidationNotifierRewardAmount` determines the fixed amount (...)
* For `notifyLiquidated`,  `liquidatedNotifierRewardAmount` determines the fixed amount (...)
```

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/146/commits/6cbea6645a620f60bf9662de09eedf531558ea46